governance/xc_admin/packages/xc_admin_common/src/governance_payload/PythGovernanceAction.ts

```diff
@@ -113,6 +113,8 @@ export class PythGovernanceHeader {
       module = MODULE_TARGET;
       action = TargetAction[this.action as keyof typeof TargetAction];
     }
+    if (toChainId(this.targetChainId) === undefined)
+      throw new Error(`Invalid chain id ${this.targetChainId}`);
```

Comment:
fwiw we should make validity-checked types for all these things so that we don't need to put checks like this everywhere. ChainName shouldn't be able to represent an invalid chain.